memgpt/local_llm/ollama/api.py

```diff
@@ -6,38 +6,36 @@
 from ..utils import count_tokens
 from ...errors import LocalLLMError
 
-HOST = os.getenv("OPENAI_API_BASE")
-HOST_TYPE = os.getenv("BACKEND_TYPE")  # default None == ChatCompletion
-MODEL_NAME = os.getenv("OLLAMA_MODEL")  # ollama API requires this in the request
```

Comment:
So once issue with the current config set up is that when the user is using local LLMs, the model is set to `local` in the agent's `config.json`:
```json
{
    "model": "local"
}
```

This is fine for most LLM backends, since you don't need to pass the model name as part of the API request, **except** for Ollama, which needs it. (Ollama is designed like a local multi-modal inference engine, so it will load/unload models depending on the `model` you specify in the POST request).

How do we want to handle this? We could add a new field called eg `model_name` that is only used if `model == "local"`:
```json
{
    "model_name": "local",
    "model": "dolphin-mistral-2.2.1"
}
```

Or, we could treat Ollama as a special case, and still require the user to user to specify the env var `OLLAMA_MODEL` when they're using it as a backend.

I guess a third option is to use a different flag / config field to distinguish between local LLMs and non-local inference. Then `model` can be left as `None` for local LLMs, or set to a real value when needed (eg with Ollama).

Reply:
@sarahwooders

Reply:
Oh this is a bug -- there should be no "local" in any config as the value of "model", and "model" and "model_wrapper" are used for ollama